manifest.konnector

```diff
@@ -1,14 +1,14 @@
 {
   "version": "1.0.0",
-  "name": "konnector name",
+  "name": "Impots.gouv.fr",
   "type": "konnector",
   "language": "node",
```

Comment:
Darty connector doesn't have the same fields... Which ones is right?

Reply:
There was a recent change in the manifest file format in forecast of store publish. The template is now update and this connector is also updated (if I did not do a mistake)